pkg/skaffold/deploy/helm.go

```diff
@@ -637,6 +630,14 @@ func envVarForImage(imageName string, digest string) map[string]string {
 		"DIGEST":     digest, // The `DIGEST` name is kept for compatibility reasons
 	}
 
+	// Standardize access to Image reference fields in templates
+	ref, err := docker.ParseReference(digest)
+	if err == nil {
+		customMap[constants.ImageRef.Repo] = ref.BaseName
+		customMap[constants.ImageRef.Tag] = ref.Tag
+		customMap[constants.ImageRef.Digest] = ref.Digest
+	}
```

Comment:
should there be a warning or debug message logged otherwise? It may be difficult for users to understand otherwise why these tags are unavailable.

Reply:
Added